app.py

```diff
@@ -1,25 +1,36 @@
-from flask import Flask, request, render_template
-import requests
+
+from flask import Flask, request, render_template , url_for , redirect  
+import requests  
+
+from markupsafe import escape
 
 app = Flask(__name__, template_folder='template')
 
 @app.route('/')
 def index():
-    return render_template("index.html")    
+    return render_template("index.html") 
+
+@app.route('/users' , methods=['POST'])     
+def welcome():
+    return "welcome"   
 
 
-@app.route('/', methods=['POST'])
+@app.route('/', methods=['POST'])           
```

Comment:
In this line, there is no change except space. Can you remove the space?

Reply:
yeah , sure .definitely I remove the space.